backend/src/secondary-adapters/cloudinary/cloudinary-container-module.ts

```diff
@@ -0,0 +1,25 @@
+import { AsyncContainerModule, interfaces } from 'inversify';
+import { CloudinaryApi, CONTAINER_TYPES } from '~/shared/types/types';
+import { v2 as cloudinary } from 'cloudinary';
+import { CONFIG } from '~/configuration/config';
+import { CloudinaryPort } from '~/core/common/port/cloudinary';
+import { CloudinaryAdapter } from './cloud/clodinary-adapter';
+
+const cloudinaryContainerModule = new AsyncContainerModule(async (bind: interfaces.Bind) => {
```

Comment:
Why do you need to use this AsyncContainerModule instead of ContainerModule. I don't see any asynchronous operations... Did I miss something?

Reply:
I guess we need since i moved whole logic concerning to cloudinary to secondary adapters
![image](https://user-images.githubusercontent.com/56759981/184027493-48e20d96-4be5-43fd-b85d-931383db4817.png)